blaze/utils.py

```diff
@@ -310,3 +310,13 @@ def __get__(self, instance, owner):
             return self
 
         return self._f(instance)
+
+
+class BlazeDeprecationWarning(DeprecationWarning):
```

Comment:
We originally thought it would be good to have a subclass to make it easily filterable, but now I'm with you, and see no really good reason not to just use `DeprecationWarning` directly.